core/pkg/opencost/allocation.go

```diff
@@ -963,17 +1037,17 @@ func (a *Allocation) RAMEfficiency() float64 {
 
 // GPUEfficiency is the ratio of usage to request. Note that, without the NVIDIA
 // DCGM exporter providing Prometheus with usage metrics, this will always be
-// zero, as GPUUsageAverage will be zero (the default value).
+// zero, as DeprecatedGPUUsageAverage will be zero (the default value).
 func (a *Allocation) GPUEfficiency() float64 {
 	if a == nil {
 		return 0.0
 	}
 
-	if a.GPURequestAverage > 0 && a.GPUUsageAverage > 0 {
-		return a.GPUUsageAverage / a.GPURequestAverage
+	if a.DeprecatedGPURequestAverage > 0 && a.DeprecatedGPUUsageAverage > 0 {
+		return a.DeprecatedGPUUsageAverage / a.DeprecatedGPURequestAverage
 	}
 
-	if a.GPUUsageAverage == 0.0 || a.GPUTotalCost() == 0.0 {
+	if a.DeprecatedGPUUsageAverage == 0.0 || a.GPUTotalCost() == 0.0 {
```

Comment:
Do we need to keep any of this logic related to the old fields? I think it would make the most sense to not have any use of the deprecated fields, and only use the new fields. If this has been done on another PR, then ignore this feedback :)

Reply:
Fixed. Used new fields.